src/components/CookieDeclaration.jsx

```diff
@@ -1,14 +1,44 @@
 import React from "react";
 
-const cookieDeclaration = ({show}) => {
-  if(show){
-    return <React.Fragment>
-            <script id="CookieDeclaration" src="https://consent.cookiebot.com/54349014-3934-489b-a905-584bc83272b2/cd.js" type="text/javascript" async></script>
-        </React.Fragment>;
+const cookieDeclaration = ({ show,handleclose }) => {
+  if (show) {
```

Comment:
Everytime show is true, it is creating this DOM?
Should it be not creating a react component and appending this logic just once?